pkgs/ops-pkg/src/genie/libs/ops/routing/iosxe/routing.py

```diff
@@ -18,67 +18,86 @@ def keys(self, item):
             return list(item.keys())
         return []
 
-    def learn(self, address_family=None, vrf=None, protocol=None, route=None, interface=None):
```

Comment:
I think no need to remove `vrf`. if None, will get all vrfs, if specified, only that vrf.

Reply:
vrf added back in

Reply:
codes looks good. but could you please check if UTs are enough to test both all VRFs and specific vrf? if not enough, please update UTs.

Reply:
@LukasMcClelland , just reminder.

Reply:
Updated tests. Ready for re-review